modules/nf-core/metamdbg/asm/main.nf

```diff
@@ -0,0 +1,62 @@
+process METAMDBG_ASM {
```

Comment:
I pushed updates to the metaMDBG module that correct languageserver errors and updates the version to 1.1